src/test/bundler_test.ts

```diff
@@ -44,12 +49,23 @@ suite('Bundler', () => {
         }
         bundler = new Bundler(bundlerOpts);
         const manifest = await bundler.generateManifest([inputPath]);
-        const documents = await bundler.bundle(manifest);
+        const bundleResult = await bundler.bundle(manifest);
+        documentBundle =
+            bundleResult.manifest.getBundleForFile(inputPath)!.bundle;
+        const {documents} = bundleResult;
         return documents.get(inputPath)!.ast;
       }
 
   suite('Default Options', () => {
 
+    test('URLs for inlined HTML imports are recorded in Bundle', async () => {
+      await bundle(inputPath);
+      assert.sameMembers(
+          [...documentBundle.inlinedHtmlImports],
+          ['imports/simple-import.html'],
+          [...documentBundle.inlinedHtmlImports].join(', '));
```

Comment:
I converted to deepEqual with a sort for the multi-member set.  Just as short as sameMembers but removes the message arguments.